examples/iris/cortex.yaml

```diff
@@ -4,28 +4,23 @@
 - kind: api
   name: tensorflow
   model: s3://cortex-examples/iris/tensorflow.zip
-  model_format: tensorflow
```

Comment:
I found a few more places where we should be able to remove `model_format`: `tutorial.md` and `packaging-models.md`